test/e2e/e2e.go

```diff
@@ -71,3 +77,21 @@ func WaitForScaleToZero(t *testing.T, deploymentName string, clients *test.Clien
 		cfg.ScaleToZeroGracePeriod*6,
 	)
 }
+
+func WaitForActivatorEndpoints(resources *v1a1test.ResourceObjects, clients *test.Clients) error {
```

Comment:
a little surprised golint didn't complain about no comment on top

Reply:
But yea, please add the comment :)

Reply:
Added doc.  Also unexport the func since it's not yet used anywhere else.